test/perf/benchmarks/assets/compiler.test.ps1

```diff
@@ -1179,7 +1179,7 @@ function Start-PSBootstrap {
                 }
             } elseif ($environment.IsLinux -and $environment.IsRedHatFamily) {
                 # Build tools
-                $Deps += "which", "curl", "gcc-c++", "cmake", "make"
+                $Deps += "which", "curl", "gcc-c++", "make"
```

Comment:
same as above

Reply:
same here. this file is for performance testing.